narwhals/_pandas_like/series.py

```diff
@@ -433,9 +433,31 @@ def is_null(self) -> PandasLikeSeries:
         ser = self._native_series
         return self._from_native_series(ser.isna())
 
-    def fill_null(self, value: Any) -> PandasLikeSeries:
-        ser = self._native_series
-        return self._from_native_series(ser.fillna(value))
+    def fill_null(
+        self,
+        value: Any | None = None,
+        strategy: Literal["forward", "backward"] | None = None,
+        limit: int | None = None,
```

Comment:
It seems that cudf does not support `limit` keyword either ([codebase](https://github.com/rapidsai/cudf/blob/branch-24.12/python/cudf/cudf/core/frame.py#L718)). I am fine with leaving the argument and everything unchanged, but the test would fail - just a heads up